runners/spark/3/src/main/java/org/apache/beam/runners/spark/structuredstreaming/SparkStructuredStreamingRunner.java

```diff
@@ -182,10 +184,7 @@ private AbstractTranslationContext translatePipeline(Pipeline pipeline) {
 
     PipelineTranslator.replaceTransforms(pipeline, options);
     prepareFilesToStage(options);
-    PipelineTranslator pipelineTranslator =
-        options.isStreaming()
-            ? new PipelineTranslatorStreaming(options)
-            : new PipelineTranslatorBatch(options);
+    PipelineTranslator pipelineTranslator = new PipelineTranslatorBatch(options);
```

Comment:
Don't believe that streaming will be supported one day in this runner? =)

Reply:
let's say with the latest databricks announcement i'm not that pessimistic anymore 🤣